GameServerLib/Game.cs

```diff
@@ -128,11 +153,12 @@ public void Initialize(Address address, string blowfishKey, Config config)
             PauseTimeLeft = 30 * 60; // 30 minutes
 
             _logger.Info("Game is ready.");
+            readyCallback();
```

Comment:
Is this ever used?

Reply:
Yeah, in GameServerApp it’s nice to know when GameServer has successfully finished. GameServerApp is a separate pull request.

Reply:
It is said to not use callbacks, but I can't think of alternate way except for events, but those are meh too.